exchanges/btcmarkets/btcmarkets_types.go

```diff
@@ -102,19 +110,159 @@ type Order struct {
 
 // TradeResponse holds trade information
 type TradeResponse struct {
-	ID           int64   `json:"id"`
-	CreationTime float64 `json:"creationTime"`
-	Description  string  `json:"description"`
-	Price        float64 `json:"price"`
-	Volume       float64 `json:"volume"`
-	Fee          float64 `json:"fee"`
+	ID           int64     `json:"id"`
+	CreationTime time.Time `json:"creationTime"`
+	Description  string    `json:"description"`
+	Price        float64   `json:"price"`
+	Volume       float64   `json:"volume"`
+	Fee          float64   `json:"fee"`
+}
+
+// AccountData stores account data
+type AccountData struct {
+	AssetName string  `json:"assetName"`
+	Balance   float64 `json:"balance,string"`
+	Available float64 `json:"available,string"`
+	Locked    float64 `json:"locked,string"`
 }
 
-// AccountBalance holds account balance details
-type AccountBalance struct {
-	Balance      float64 `json:"balance"`
-	PendingFunds float64 `json:"pendingFunds"`
-	Currency     string  `json:"currency"`
+// TradeHistoryData stores data of past trades
+type TradeHistoryData struct {
+	ID            string  `json:"id"`
+	MarketID      string  `json:"marketId"`
+	Timestamp     string  `json:"timestamp"`
+	Price         float64 `json:"price,string"`
+	Amount        float64 `json:"amount,string"`
+	Side          string  `json:"side"`
+	Fee           float64 `json:"fee,string"`
+	OrderID       string  `json:"orderId"`
+	LiquidityType string  `json:"liquidityType"`
+}
+
+// OrderData stores data for new order created
+type OrderData struct {
+	OrderID      string    `json:"orderId"`
+	MarketID     string    `json:"marketId"`
+	Side         string    `json:"side"`
+	Type         string    `json:"type"`
+	CreationTime time.Time `json:"creationTime"`
+	Price        float64   `json:"price,string"`
+	Amount       float64   `json:"amount,string"`
+	OpenAmount   float64   `json:"openAmount,string"`
+	Status       string    `json:"status"`
+}
+
+// CancelOrderResp stores data for cancelled orders
+type CancelOrderResp struct {
+	OrderID       string `json:"orderId"`
+	ClientOrderID string `json:"clientOrderId"`
+}
+
+// PaymentDetails stores payment address
+type PaymentDetails struct {
+	Address string `json:"address"`
```

Comment:
This will not always be address
[List deposits/withdrawals ](https://api.btcmarkets.net/doc/v3#tag/Fund-Management-APIs/paths/~1v3~1transfers/get)

They also currently use "txId as a possible return field